### ISSUE TITLE
Cannot decode overview_polyline

### ISSUE THREAD

**Issue by yizhexu** [SymbolixAU/googleway#33]
I try to decode the overview polyline from this request and got cannot be decoded error:
"https://maps.googleapis.com/maps/api/directions/json?origin=place_id:ChIJg0QqCUx-bIcRTdSPo8yl9XA&destination=39.763262,-105.143201&mode=driving&units=imperial&departure_time=1482433621&traffic_model=pessimistic&avoid=highway&key=[KEY]"

aweqF~~v_SgVGAoM@iFaT@oEAoAA}@?oBzBqHpI}@bAWf@MTe@p@{H~JkBlC]v@kAlAgGnHmInKwEtGeA|Aq@pAWn@k@jBy@tDeBhIq@`D]jAm@lB_@|@_A~AmAdBw@|@oAbAsA|@kBt@kBn@aHbCcFlBcHjDqCvA_HrDaAb@yBnA_E`CiBfBs@x@{@nAiAlBkD~FmAlBw@dA{ClDsHhIiDzDSVuA|B{AbCiBbCcEdEmAdAmBrAiDrBoChAeCx@yGjBkCr@yEbAaAL_APs@Ju@@}A@sACoAIcAOcB_@cA[{By@{BaAiCw@}Ds@s@EeAIsCCwA?mFHE?i@FgBZqA`@gAh@YNUNuBfBwC`D_B`BcBbBqBbBwAv@_Bf@eARcALyABoBI{A[cD_A_D}@wBq@aEyAmAc@c@SiBgAaAq@eEcEaB{BsB{CwA{Bs@gAwNyT_FwHuHwMkAqBeBsCaBoBmAkAcBwA}B{AkBoA_@_@aC}A_Aq@q@k@_AcAmAsBe@aAiAiDk@aB]wAaDwKa@yA[cAOe@BG?M?ICMuAwEk@oAy@uA]i@qAwAMQCIQO_DeC}CeBgBw@mBm@oA]aB[eC[sCSwCCaMEk@GiAWm@IoDoBw@Ym@G_ABm@TMJg@^y@fASl@Ib@EPCZC`@Cf@DzF@~A@~@@xBC~CQfC[rBsAtHo@|DaAxGGd@?j@wAfHeAbGWzBI`DFhBRxCf@`HHdCChBs@hQQdGE~MGlMEfXUvDa@tDq@bHGhAKbE?hBHbFP`Fb@`HjAlNZdDNdCB~BExBEfBS`IChF?rM?jUGbCIh@Q~Aa@hB{@~BmBjF[pAObAStBCzCB~@`@pD^`B|@hCnAxCt@vBRt@XbBLtAFdBBnM@fb@?bB[~TIdCGbA]rCk@zCYfA{@fC{@rBo@jA_B|B{BpCyAxBe@|@i@bBe@rBQfBElBBlBLpB`@rDpC|Qb@pCvA`Kb@~CpBfNbBzKz@~FXnCDt@JzBDpBHlP@bd@AfJObE}@fOU`FCrE@td@Fll@F|FPnFZfFb@pEz@vGn@pDp@~CnJt^pHrXrA`FdAjD`@dAf@`AnAjBt@bApAlAx@h@^T~At@pA`@`ARvALlDDhDCnXEff@O

It can be decoded with [https://developers.google.com/maps/documentation/utilities/polylineutility](polyline utility tool).

**Comment by SymbolixAU** [comment]
I'm not getting an error decoding that string. 

```

pl <- "aweqFv_SgVGAoM@iFaT@oEAoAA}@?oBzBqHpI}@bAWf@MTe@p@{HJkBlC]v@kAlAgGnHmInKwEtGeA|Aq@pAWn@k@jBy@tDeBhIq@D]jAm@lB_@|@_A~AmAdBw@|@oAbAsA|@kBt@kBn@aHbCcFlBcHjDqCvA_HrDaAb@yBnA_ECiBfBs@x@{@nAiAlBkDFmAlBw@dA{ClDsHhIiDzDSVuA|B{AbCiBbCcEdEmAdAmBrAiDrBoChAeCx@yGjBkCr@yEbAaAL_APs@Ju@@}A@sACoAIcAOcB_@cA[{By@{BaAiCw@}Ds@s@EeAIsCCwA?mFHE?i@FgBZqA@gAh@YNUNuBfBwCD_BBcBbBqBbBwAv@_Bf@eARcALyABoBI{A[cD_A_D}@wBq@aEyAmAc@c@SiBgAaAq@eEcEaB{BsB{CwA{Bs@gAwNyT_FwHuHwMkAqBeBsCaBoBmAkAcBwA}B{AkBoA_@_@aC}A_Aq@q@k@_AcAmAsBe@aAiAiDk@aB]wAaDwKa@yA[cAOe@BG?M?ICMuAwEk@oAy@uA]i@qAwAMQCIQO_DeC}CeBgBw@mBm@oA]aB[eC[sCSwCCaMEk@GiAWm@IoDoBw@Ym@G_ABm@TMJg@^y@fASl@Ib@EPCZC@Cf@DzF@A@@@xBCCQfC[rBsAtHo@|DaAxGGd@?j@wAfHeAbGWzBIDFhBRxCf@HHdCChBs@hQQdGEMGlMEfXUvDa@tDq@bHGhAKbE?hBHbFPFb@HjAlNZdDNdCBBExBEfBSIChF?rM?jUGbCIh@Q~Aa@hB{@~BmBjF[pAObAStBCzCB~@@pD^B|@hCnAxCt@vBRt@XbBLtAFdBBnM@fb@?bB[~TIdCGbA]rCk@zCYfA{@fC{@rBo@jA_B|B{BpCyAxBe@|@i@bBe@rBQfBElBBlBLpB@rDpC|Qb@pCvAKb@~CpBfNbBzKz@~FXnCDt@JzBDpBHlP@bd@AfJObE}@fOUFCrE@td@Fll@F|FPnFZfFb@pEz@vGn@pDp@~CnJt^pHrXrAFdAjD@dAf@AnAjBt@bApAlAx@h@^T~At@pA@`ARvALlDDhDCnXEff@O"
df <- decode_pl(pl)
head(df)
#        lat      lon
# 1 39.68385 -0.10252
# 2 39.68757 -0.10248
# 3 39.68758 -0.10016
# 4 39.68757 -0.09899
# 5 39.69094 -0.09900
# 6 39.69198 -0.09899
```

however, by re-running the query I am geting different values for 'lon'. 

```
res <- google_directions(origin = "place_id:ChIJg0QqCUx-bIcRTdSPo8yl9XA",
                         destination = c(39.763262,-105.143201),
                         mode = "driving",
                         units = "imperial",
                         departure_time = Sys.time() + 60,
                         traffic_model = "pessimistic",
                         avoid = "tolls",
                         key = key
                         )

newPl <- res$routes$overview_polyline$points

head(decode_pl(newPl))
#        lat       lon
# 1 39.68385 -104.9805
# 2 39.68757 -104.9804
# 3 39.68758 -104.9781
# 4 39.68757 -104.9769
# 5 39.69094 -104.9770
# 6 39.69198 -104.9769

```

**Comment by SymbolixAU** [comment]
I'm not sure the 'copy & pasting' of the string is working as some of the characters are getting changed. Are you doing anything with the polyline after running the query, or are you calling `decode_pl()` on the result directly (as I have done in my example)?

**Comment by yizhexu** [comment]
Thank you for your input. I just tested it again and found out the issue I got was because some setting in Rstudio. 

In detail, I used the following function to conduct the api call
`connect <- url(URLSTRING); on.exit(close(connect), add = TRUE)
lines <- try(readLines(connect, encoding="UTF-8", warn = FALSE), silent = TRUE)
route <- jsonlite::fromJSON(paste(lines, collapse = ""))
rcpp_decode_pl(route$routes$overview_polyline$points)
`

I found out the error I got was because by default, I have Rstudio setting "limit length of lines displayed in console to 1000" which somehow truncated my readLines output. That caused the code not working. 

Thank you for your quick response. I'm sorry for the confusion.
